src/karma-webpack.js

```diff
@@ -242,7 +242,7 @@ Plugin.prototype.make = function(compilation, callback) {
     var entry = file
 
     if (this.wrapMocha) {
-      entry = `${require.resolve('./mocha-env-loader')}!${entry}`
+      entry = `${require.resolve('./mocha-env-loader')}?name=${compilation.name}!${entry}`
```

Comment:
Could this be potentially breaking ? It's fine since `v4.0.0` is a `beta` release, but we need to be aware just in case :)

Reply:
I've just tested it out and this will need to drop support for anything less than webpack 4 as it turns out `webpack-dev-middleware` has.

Reply:
I'm missing the context here, this change doesn't seemed to be related to `webpack-dev-middleware`(?), please elaborate here :). If you mean the current version of `webpack-dev-middleware` shipping with `karma-webpack@4.0.0-beta.0` then we need to fix the [`peerDependencies`](https://github.com/webpack-contrib/karma-webpack/blob/next/package.json#L41)

Reply:
Updated `peerDependencies`, yeah `karma-webpack@4.0.0-beta.0` doesn't work with `webpack < v4.0.0`.